Cargo.toml

```diff
@@ -133,7 +133,7 @@ required-features = ["display"]
 
 [[test]]
 name = "error"
-path = "tests/error.rs"
+path = "tests/error_tests.rs"
```

Comment:
Had to change file name, as I've moved tests to separate module and Rust won't allow me to have `mod error` in `error.rs`.